index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="Home">
+    <div class=" flex nav-bar">
+        <a href="#Home"><img src="./images/gtco_logo2.png" alt="Logo"></a>
+        <ul class="flex nav-list-item">
+            <li class="nav-item nav-about"><a href="about.html">About</a></li>
+            <li class="nav-item nav-color"><a href="#">Exhibition</a></li>
+            <li class="nav-item nav-color"><a href="#">Join</a></li>
+            <li class="nav-item nav-color"><a href="#">Sponsor</a></li>
+            <li class="nav-item nav-color"><a href="#">Schedules</a></li>
+            <li class="nav-item nav-border"><a href="#program">THE CHEFS</a></li>
+        </ul>
+    </div>
+    </header>
```

Comment:
I suggest you add the top navigation as in the design by cindy with the social media links in your desktop version. Kindly note that the header has two nav bars:
<img width="848" alt="Screenshot 2022-10-29 at 17 11 01" src="https://user-images.githubusercontent.com/48546992/198836727-9c74aeba-2cbb-4a30-b434-5af10b446e71.png">